policy/release/cve.rego

```diff
@@ -141,17 +141,79 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [])
 }
 
+# METADATA
+# title: Deprecated task name
+# description: >-
+#   The `CLAIR_SCAN_RESULT` task name has been deprecated, and has been
+#   replaced with `SCAN_OUTPUT`. If any task results with the old name are
+#   found, this rule will raise a warning.
+# custom:
+#   short_name: deprecated_cve_result_name_warning
+#   failure_msg: CVE scan uses deprecated task name
+#   solution: >-
+#     Use the newer `SCAN_OUTPUT` task name.
+#   collections:
+#   - minimal
+#   - redhat
+#   depends_on:
+#   - attestation_type.known_attestation_type
+#
+warn contains result if {
+	_vulnerabilities_deprecated
+	result := lib.result_helper(rego.metadata.chain(), [])
+}
+
+# METADATA
+# title: Deprecated task name for unpatched vulnerabilities
```

Comment:
All comments left for the policy rule above apply here as well.

Reply:
Done!